<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>351</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="metinText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Lütfen metin dosyasını yükleyiniz:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ozetText">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>50</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Lütfen özet dosyasını yükleyiniz:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="nodeSentence">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
